fabric/Makefile

```diff
@@ -19,18 +19,19 @@ build: native
 patch: $(FABRIC_PATCHED)
 
 $(FABRIC_PATCHED):
-	@echo "Patching Fabric ..."
-	@if [ "$(FABRIC_CURRENT_TAG)" != "$(FABRIC_VERSION)" ]; then \
-		echo "Aborting! Tag on current HEAD ($(FABRIC_CURRENT_TAG)) does not match expected tag/$(FABRIC_VERSION)!"; exit 1; \
+	echo "Patching Fabric ..."; \
```

Comment:
I would bring the @echo back to avoid output the entire if statement the console